src/javaharness/java/arcs/webimpl/WebHarnessModule.java

```diff
@@ -7,15 +7,17 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import javax.inject.Singleton;
 
 @Module
 public abstract class WebHarnessModule {
 
   @Binds
   public abstract ArcsEnvironment provideStandaloneWebArcsEnvironment(ShellApiBasedArcsEnvironment impl);
 
+  @Singleton
```

Comment:
Similar to above comment -- why does it need to be `Singleton` now? Just want to understand backstory.

Reply:
(this is part of the same change, replied above)